docs/pages/guides/server-components.mdx

```diff
@@ -37,9 +35,7 @@ Enable the flag in the project Expo config:
 }
 ```
 
-Ensure the entry module is `expo-router/entry` (default).
-
-Create an initial route `app/index.tsx`:
+Ensure the entry module is `expo-router/entry` (default) in **package.json**. Then, create an initial route **app/index.tsx**:
```

Comment:
```suggestion
3. Create an initial route **app/index.tsx**:
```

We can remove the context of entry module since that is now covered in the step 1.